docs/05-technical-reference/ac-01-application-gateway-details.md

```diff
@@ -44,3 +44,9 @@ Application Gateway proxies the following headers while making calls to the regi
 - `X-Forwarded-Client-Cert`
 
 In addition, the `User-Agent` header is set to an empty value not specified in the call, which prevents setting the default value.
+
+## Response rewriting
+
+If during a call to the external system, the target responds with a redirect (`3xx` status code) that points to the URL with the same host and a different path, the `Location` header is modified so that the original target path is replaced with the Application Gateway URL and port, with the sub-path pointing to the called service attached at the end, in this format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
+
+This functionality makes the HTTP clients that originally called Application Gateway follow redirects through the Gateway, and not to the service directly. This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
```

Comment:
```suggestion
This functionality makes the HTTP clients that originally called Application Gateway follow redirects through the Gateway, and not to the service directly. This allows for passing authorization, custom headers, URL parameters, and the body.
```
I'd drop the `without an issue` part. It sounds like sth could go wrong. We don't want that :D

Reply:
It could go wrong if there is no response rewriting.